server/lemmy_utils/src/settings.rs

```diff
@@ -123,7 +123,11 @@ impl Settings {
   }
 
   pub fn get_config_defaults_location() -> String {
-    env::var("LEMMY_CONFIG_LOCATION").unwrap_or_else(|_| CONFIG_FILE_DEFAULTS.to_string())
+    env::var("LEMMY_CONFIG_DEFAULTS_LOCATION").unwrap_or_else(|_| CONFIG_FILE_DEFAULTS.to_string())
```

Comment:
I renamed this env var, which is backwards-incompatible. It seemed confusing though to have `CONFIG_FILE` not be equivalent to `LEMMY_CONFIG_LOCATION`. Is there a better name that I should use down below for the equivalent of `CONFIG_FILE`?

Reply:
That change seems good to me. Make sure you grep the codebase when making changes tho, I found an instance of this env var in the docs at `docs/src/administration_configuration.md`